test/agents/test_chat_agent.py

```diff
@@ -72,13 +72,12 @@ def test_chat_agent(model: ModelType):
 @pytest.mark.parametrize('n', [1, 2, 3])
 def test_chat_agent_multiple_return_messages(n):
     model_config = ChatGPTConfig(temperature=1.4, n=n)
-    system_msg = SystemMessage("Assistant", RoleType.ASSISTANT,
-                               content="You are a helpful assistant.")
+    system_msg = BaseMessage("Assistant", RoleType.ASSISTANT, meta_dict=None,
```

Comment:
sometimes meta_dict is initialized as None, sometimes it's an empty dict. Why is it so?

Reply:
Good catch. I also produced a drop of tears looking at this inconsistency. I would clean up this in a follow up PR. This PR just keeps this inconsistency as is. I am still yet to learn the purpose of the meta_dict.

Reply:
A direct answer is: historical inconsistency. Not a big deal IMO.